vyper/context/types/function.py

```diff
@@ -199,8 +239,15 @@ def from_FunctionDef(
             else:
                 raise StructureException("Bad decorator syntax", decorator)
 
-        if "is_public" not in kwargs:
-            raise FunctionDeclarationException("Visibility must be public or private", node)
+        if "function_visibility" not in kwargs:
+            options = " or ".join([v.value for v in list(FunctionVisibility)])
```

Comment:
join can accept an iterator, no need for the square brackets here

Reply:
```suggestion
            options = " or ".join(v.value for v in list(FunctionVisibility))
```

Reply:
:+1: